### ISSUE TITLE
Quickchatbar always hidden on startup since last snapsho

### ISSUE THREAD

**Issue by Kissaki** [mumble-voip/mumble#2572]
Since upgrading to Snapshot 1634 the quickchat input is always resized to 0 height.

Potentially related recent commits:
* https://github.com/mumble-voip/mumble/commit/4009ea353944fc9cbefcbf95979f0d6d0c0ec3b2
* https://github.com/mumble-voip/mumble/commit/99899d12decccb5670d4e6887c4f7ec7f3cf8faf

When I mentioned this issue in (our internal) chat, someone else mentioned him experiencing it as well.

**Comment by Tarun80** [comment]
I've had this issue as of 1634 along with several of my friends who use the snapshots (same version, 1634).

**Comment by SuperTux88** [comment]
I have the same issue on linux since a long time (probably since the same change broke it on windows) and I just found this issue. It says here that it's fixed, but it was only fixed for windows, not for linux, because this issue was marked was "windows only". I didn't find a second issue for the same problem for linux.

The same fix that worked for windows also worked for me on linux, and the quickchatbar is now visible from the start on. But fixing it on linux as well is basically undoing the commit that initially broke it:

```diff
diff --git a/src/mumble/MainWindow.cpp b/src/mumble/MainWindow.cpp
index db67de48..32460b3f 100644
--- a/src/mumble/MainWindow.cpp
+++ b/src/mumble/MainWindow.cpp
@@ -353,7 +353,7 @@ void MainWindow::setupGui()  {
 
        updateTransmitModeComboBox();
 
-#ifdef Q_OS_WIN
+#ifndef Q_OS_MAC
        setupView(false);
 #endif
 
```

That fixes the quickcharbar for me on both 1.3.0 and `master`, without getting the original problem that was fixed in #2552 back. So I have the chatbar, but the MainWindow renders perfectly fine. But I don't understand what the original problem was, since it only says "certain Qt 5 versions" and I don't know which Qt5 versions or what exactly was wrong with these Qt5 versions, and I don't know if I'm just lucky and don't have an affected Qt5 version and the problem still exists with other Qt5 versions, or if the problem was fixed and doesn't exist anymore. That's why I'm not going to just create a PR with the patch above since I don't know what exactly I need to test to make sure the original bug doesn't appear again.

Does anybody understand what the original problem from #2192 was and why disabling the `setupView(false);` line helped (which broke the quickchatbar)? Is there maybe another fix that doesn't break the quickchatbar? Or is the fix not needed anymore?
